Overview/index.html

```diff
@@ -0,0 +1,4 @@
+---
+title: Empty document to protect directory
+layout: default
+---
```

Comment:
GitHup complains about "non newline at end of file"

Reply:
Added a newline.